lua/entities/gmod_wire_pod.lua

```diff
@@ -442,6 +442,10 @@ function ENT:Think()
 						button.EntToOutput = ply
 						button:Switch( true )
 					end
+				elseif button.Use then
+					-- Generic support (EGP screens, etc)
+					self.MouseDown = true
+					button:Use(ply, ply, USE_ON, 0)
```

Comment:
The second argument here should be `self`, not `ply`. From the Valve developer wiki:
> !activator
    The entity that began the current I/O chain. If a player walks into a trigger that fires a logic_relay, the player is the !activator of the relay's output(s). 
!caller
    The previous entity in the current I/O chain. If a player walks into a trigger that that fires a logic_relay, the trigger is the !caller of the relay's output(s).

This would then allow the button to detect itself whether `self.podpress` should be true or not (by seeing whether the player pressed the button via a pod controller).